### DISCUSSION TITLE
How to delete gameplay mod presets on touch screen/mobile?

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#23303] [Q&A] [answered]
On a computer you can right click a personal preset and click delete, but mobile doesn't have right click. Long press doesn't work either.

**Comment by peppy** [comment, marked as answer]
Long press for right click support is planned (and already implemented, pending review). Please wait a bit for that to be available.

**Reply by Theighlin** [reply]
Implemented via https://github.com/ppy/osu-framework/pull/5255
